ArduCopter/mode_guided.cpp

```diff
@@ -853,6 +903,28 @@ void ModeGuided::velaccel_control_run()
     attitude_control->input_thrust_vector_heading(pos_control->get_thrust_vector(), auto_yaw.get_heading());
 }
 
+// checks for incoming actuator commands
+void ModeGuided::motor_control_run()
+{
+#if FRAME_CONFIG == MULTICOPTER_FRAME
+    // set actuator values to zero if no updates received for 3 seconds
+    uint32_t tnow = millis();
+    if (tnow - guided_motor_state.update_time_ms > get_timeout_ms()) {
```

Comment:
```suggestion
    if (millis()- guided_motor_state.update_time_ms > get_timeout_ms()) {
```

No need for the local if you only use it once.

Reply:
Done